utils/build.gradle

```diff
@@ -6,12 +6,15 @@ dependencies {
     api project(":annotations")
     api project(":pluginapi")
     api project(":utils:reflector")
-    api "com.google.auto.service:auto-service:1.0-rc4"
     api "javax.inject:javax.inject:1"
     api "javax.annotation:javax.annotation-api:1.3.2"
+
+    // For @VisibleForTesting
+    api "com.android.support:support-annotations:28.0.0"
```

Comment:
Not totally sure, but `@VisibleForTesting` is `RententionPolicy.CLASS`, so it technically cannot be accessed through reflection, thus should not neccessarily be part of the API, or am I wrong here?

Reply:
Was matching their convention with other annotations artifacts above